.yeoman/templates/containers/README.md

```diff
@@ -0,0 +1,53 @@
+# psammead-<%= name %> - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-<%= name %>%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-<%= name %>%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-<%= name %>)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-<%= name %>) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-<%= name %>)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-<%= name %>&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/<%= name %>--containing-image) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-<%= name %>.svg)](https://www.npmjs.com/package/@bbc/psammead-<%= name %>) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `<%= componentName %>` component does...
```

Comment:
Should this be 
```suggestion
The `<%= containerName %>` container does...
```

Reply:
I think the `componentName` should remain as is, unless there's a difference in the way it's formatted.

But the `component` text can be changed to `container` for more clarity

Reply:
Fixed!